liquibase-integration-tests/src/test/java/liquibase/dbtest/h2/H2IntegrationTest.java

```diff
@@ -32,10 +33,9 @@ public H2IntegrationTest() throws Exception {
         this.dbmsExcludeChangelog = "changelogs/h2/complete/dbms.exclude.changelog.xml";
     }
 
-    @Override
-    protected boolean isDatabaseProvidedByTravisCI() {
-        // H2 is an in-process database
-        return true;
+    @Test
+    public void showVersion() throws DatabaseException {
+        System.out.println(getDatabase().getDatabaseProductVersion());
```

Comment:
Does this test do anything?

Reply:
No, it was something I added to see how I could change versions through settings. I'll take it out.